cmd/serve/main.go

```diff
@@ -32,8 +33,15 @@ func main() {
 	printRules(router.Rules)
 
 	port := config.ServerPort
+	server := &http.Server{
+		Addr:              ":" + port,
```

Comment:
Do we want to listen on all interfaces? If not we might want to switch to `127.0.0.1`

> If host is omitted, as in ":8080", Listen listens on all available interfaces instead of just the interface with the given host address. See Dial for more details about address syntax.
https://pkg.go.dev/net#Listen

Reply:
I'm pretty sure that at least 0.0.0.0 would be needed for this to work in a container. Yes, the api gateway could be used elsewhere but for now it isn't. We could make this configurable but I'd say that's out of the scope for this PR.